src/components/Layout/Header/Header.module.scss

```diff
@@ -0,0 +1,45 @@
+@import '../../../variables.scss';
+
+*{
+    padding: 0;
+    margin: 0;
+}
+
```

Comment:
This should be added in separate file named `global.scss` and imported in `index.tsx`. All `.module` files should only style component that they are tied to.